public/static/locales/th/common.json

```diff
@@ -45,16 +45,20 @@
     "school": {
       "title": "แอพโรงเรียน",
       "mainSite": "เว็บหลัก",
-      "kularbFah": "กุหลาบฟ้า",
-      "kularbLuang": "กุหลาบหลวง",
+      "samak": "สมัคร",
       "shopping": "SK Shopping",
+      "schoolICT": "School ICT",
```

Comment:
Keeping the school ICT text organized in this way could be better than leaving it in "other".